docs/source/operations/adding_new_org_fabric.md

```diff
@@ -61,7 +61,7 @@ For reference, see `network-fabric-add-organization.yaml` file [here](https://gi
 The [add-new-organization.yaml](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/shared/configuration/add-new-organization.yaml) playbook is used to add a new organization to the existing network. This can be done using the following command
 
 ```
-ansible-playbook platforms/shared/configuration/add-new-organization.yaml --extra-vars "@path-to-network.yaml"
+ansible-playbook platforms/shared/configuration/add-new-organization.yaml --extra-vars "@path-to-network.yaml" -e "add_new_org=true"
```

Comment:
add_new_org is not needed when playbook is add-new-org

Reply:
My mistake, it is only needed when calling the `platforms/hyperledger-fabric/configuration/add-organization.yaml` directly. I've changed this in commit [95916d1](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1096/commits/95916d1d21a7b0cc4ed0c8ac7087c1db6ad2ce0f)